foxtrot-core/src/main/java/com/flipkart/foxtrot/core/reroute/ShardInfo.java

```diff
@@ -0,0 +1,19 @@
+package com.flipkart.foxtrot.core.reroute;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.elasticsearch.index.shard.ShardId;
+
+/***
+ Created by mudit.g on Feb, 2019
+ ***/
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class ShardInfo {
```

Comment:
Why can't shardid be used directly? why wrap a single element in another class?

Reply:
This was added a placeholder to enrich UI for adding more details around the shard information